CHANGELOG.md

```diff
@@ -68,9 +68,10 @@
 - `pd.read_csv` reads using the native pandas CSV parser, then uploads data to snowflake using parquet. This enables most of the parameters supported by `read_csv` including date parsing and numeric conversions. Uploading via parquet is roughly twice as fast as uploading via CSV.
 - Initial work to support an Index directly in Snowpark pandas. Support for Index as a first-class component of Snowpark pandas is coming soon.
 - Added lazy index constructor and support for `len`, `shape`, `size`, `empty`, `to_pandas()` and `names`. 
-- Added support for `Index.copy()`
 - For `df.index`, Snowpark pandas creates a lazy index object. 
 - For `df.columns`, Snowpark pandas supports a non-lazy version of an Index since the data is already stored locally
+- Added support for `Index.copy()`
```

Comment:
should not move this line

Reply:
wouldn't it be better if the APIs mentioned were all at the bottom?

Reply:
Then why not combine into one line?

Reply:
Because they are separate PRs, but I can combine if you think that is better

Reply:
There are two options:
1. keep it as is for what happened out side of this PR. Because when we release, we will visit this and reorganize it later
2. If you want to update both make sure update them correctly. E.g., line 74 does not say what support are they? Are they the APIs for series, df, or index?

Reply:
Ok I will update it to be better

Reply:
IMO this should be combined into one line, the release notes are customer facing so condensing it is better. You could even say something like "Added support for the following `Index` APIs: `values`, `item()`, etc.